libraries/AP_Scripting/docs/docs.lua

```diff
@@ -541,6 +551,14 @@ function SocketAPM_ud:listen(backlog) end
 ---@return integer
 function SocketAPM_ud:send(str, len) end
 
+-- send a lua string to a specified address. May contain binary data
+---@param str string
+---@param len uint32_t_ud|integer|number
+---@param ipaddr uint32_t_ud -- ipv4 address
```

Comment:
```suggestion
---@param ipaddr uint32_t_ud|integer|number -- ipv4 address
```
All uint32 params can do this magic casting.

Reply:
changed